### ISSUE TITLE
Making sure the denoiser has a element_mask_func

### ISSUE THREAD

**Issue by swagataroy123** [microsoft/mattergen#85]
I checked that the denoiser does not mask disallowed element by default. IS there a way to pass the argument to mask the disallowed logits?

**Comment by danielzuegner** [comment]
Hi @swagataroy123,

you can modify the model config in [`mattergen.yaml`](https://github.com/microsoft/mattergen/blob/main/mattergen/conf/lightning_module/diffusion_module/model/mattergen.yaml):

```yaml
_target_: mattergen.denoiser.GemNetTDenoiser
...
# add this:
element_mask_func: 
  _target_: mattergen.denoiser.mask_disallowed_elements
  _partial_: true
```

This will enable the logit masking. Let me know if anything remains unclear.
